lib/rex/text/rand.rb

```diff
@@ -258,5 +258,27 @@ def self.rand_mail_address
       mail_address << '@'
       mail_address << Rex::Text.rand_hostname
     end
+
+    # Generate a strong password of specified length and attributes
+    def self.rand_password(len=10, uppercase=true, lowercase=true, numbers=true, special=false)
```

Comment:
```suggestion
    def self.rand_password(len=10, mix_case: true, special_characters: false)
```

Using named arguments would allow users to specify arbitrary constraints in any order:
```ruby
::Rex::Text.rand_password(special_characters: true)
```

The existing approach makes that more awkward:
```ruby
::Rex::Text.rand_password(true, true, true, true)
```